src/components/User/README.md

```diff
@@ -1,14 +1,89 @@
-## User
+<!--GITHUB_BLOCK-->
 
-Display user avatar and his brief info.
+# User
 
-### PropTypes
+<!--/GITHUB_BLOCK-->
 
-| Name        | Type             | Required | Default | Description                                                                                                     |
-| :---------- | :--------------- | :------- | :------ | :-------------------------------------------------------------------------------------------------------------- |
-| imgUrl      | `string`         |          |         | Url of user avatar                                                                                              |
-| className   | `string`         |          |         | Root element class name                                                                                         |
-| name        | `string`         |          |         | User name (first line of info)                                                                                  |
-| description | `string`         |          |         | User additional data (second line of info)                                                                      |
-| size        | `UserAvatarSize` |          | 'm'     | Component size. Supported values is: `xs`, `s`, `m`, `l`, `xl`. With a smallest size user info is not rendered. |
-| qa          | `string`         |          |         | HTML `data-qa` attribute, used in tests                                                                         |
+```tsx
+import {User} from '@gravity-ui/uikit';
+```
+
+General component for displaying a user avatar with a info block. It uses [Avatar](https://gravity-ui.com/components/uikit/avatar) component to render the avatar and based on [UserWrapper](https://gravity-ui.com/components/uikit/user-wrapper).
```

Comment:
The links can be to the README files, they will be converted to the normal URLs on render